packages/base-driver/test/basedriver/driver-e2e-tests.js

```diff
@@ -464,7 +467,7 @@ function baseDriverE2ETests (DriverClass, defaultCaps = {}) {
           return {;
         `;
         const {data} = await axios({
-          url: `http://localhost:${port}/session/${sessionId}/appium/execute_driver`,
+          url: `http://${TEST_HOST}:${port}/session/${sessionId}/appium/execute_driver`,
```

Comment:
I would rather create a small function to make such urls, where we pass port number, path and session Id. It could even be curried for particular suites, so less arguments are needed to be passed